src/components/ToDoItem.vue

```diff
@@ -1,3 +1,87 @@
-<template></template>
+<template>
+  <div
+    class="relative sm:mx-auto mx-4 flex sm:flex-col space-y-8 justify-center rounded-2xl bg-white border-black border-2 p-[18px] m-[50px] sm:w-[610px] w-80 flex-row"
+  >
+    <div class="w-full flex flex-row">
+      <div class="m-auto mr-5">
+        <div
+          class="p-2 border-black border-[3px] rounded-full w-6 h-6 flex-row lg:hidden"
+        ></div>
+      </div>
+      <div class="flex flex-col w-full items-center">
+        <p
+          class="w-full placeholder-black sm:text-5xl text-lg flex font-bold text-area lg:flex mt-5"
+        >
+          Title
+          {{ toDo.title }}
+        </p>
+        <div
+          class="flex w-full items-center sm:h-[12px] h-[5px] lg:flex hidden"
+        >
+          <img :src="dateIcon" class="lg:flex hidden mr-1 h-3 w-3" />
+          {{ newDate() }}
+        </div>
 
-<script></script>
+        <div class="flex w-full sm:h-[12px] lg:hidden text-gray-500">
+          <img :src="dateIcon" class="mr-1 h-3 w-3 hidden" />
+          {{ newDate() }}
+        </div>
+      </div>
+
+      <div class="flex">
+        <div
+          class="sm:invisible visible rounded-full h-2.5 w-2.5 my-9"
+          :class="{
+            'bg-green-500': 'Low' === localTodo.priority,
+            'bg-yellow-500': 'Medium' === localTodo.priority,
+            'bg-red-500': 'High' === localTodo.priority,
+          }"
+        ></div>
+        <ToDoPriority v-model:priority="localTodo.priority" />
+      </div>
+    </div>
+
+    <div class="flex w-full justify-between hidden">
+      <div>
+        <p
+          class="w-full placeholder-black leading-[3rem] sm:text-5xl text-lg flex mt-4 font-bold text-area lg:flex hidden"
+        >
+          Title
+        </p>
+      </div>
+
+      <div class="hidden">
+        <ToDoPriority v-model:priority="localTodo.priority" />
+      </div>
+      <div class="flex w-full items-center sm:h-[12px] h-[5px] lg:flex hidden">
+        <img :src="dateIcon" class="lg:flex hidden mr-1 h-3 w-3" />
+        {{ newDate() }}
+      </div>
+    </div>
+
+    <div
+      class="w-full p-2 flex flex-col justify-between text-xl break-all text-gray-500 sm:text-3xl font-semibold relative hidden lg:flex"
+    >
+      <div class="flex justify-between">
+        <div class="sm:text-3xl text-base flex-wrap break-all lg:flex hidden">
+          <p placeholder="">Cras placerat proin.</p>
+        </div>
+
+        <div class="p-2 border-black border-[6px] rounded-full w-10 h-10"></div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+  import { ref } from 'vue';
+  import ToDoPriority from './ToDoPriority.vue';
+  import dateIcon from './../assets/dateIcon.svg';
+  import { newDate } from '../utils/date';
+  const priorities = ['Low', 'Medium', 'High'];
```

Comment:
Please set the priorities as a const variable because this list is set here and in the TodoItemEdit.vue as well. It is used in the template as well. 
In case of there is a list, its items are the same and it is used in more cases, it is better to create a const, set the value (ex. 'Low') and use it everywhere to avoid typos. Also if you have to update the values or extend the list, you have to do it at one place only.